api/user.ts

```diff
@@ -56,6 +56,12 @@ export function validBio(bio: string): boolean {
   return bio.length <= 200;
 }
 
+export enum CreateUserError {
+  InvalidUsername = 'Invalid Username! Please choose a valid username',
+  InvalidDisplayName = 'Invalid Display Name! Please choose a valid display name',
+  UsernameTaken = 'Username Taken! Please choose another username',
+}
```

Comment:
IMO all info about what's acceptable/valid should already be on front-end & stay there until we get past the create user screen, so error messages like this are good